karma/module.py

```diff
@@ -407,15 +407,31 @@ async def karma_vote(
 
         # Fetch updated message with the votes
         vote_message = await vote_message.channel.fetch_message(vote_message.id)
+        total_reactions = 0
+
         for reaction in vote_message.reactions:
             votes[reaction.emoji] = reaction.count - 1
+            total_reactions += reaction.count - 1
 
         log_message: str = (
             f"Karma vote over emoji '{emoji_name}' ended: "
             + ", ".join(f"{v}x {emoji_labels[k]}" for k, v in votes.items())
             + "."
         )
 
+        if total_reactions < voter_limit:
+            await guild_log.info(
+                ctx.author,
+                ctx.channel,
+                _(ctx, log_message + " Not enough voters, aborted."),
+            )
+            await ctx.send(
+                _(ctx, "Vote over {emoji} failed - not enough voters.").format(
```

Comment:
Well, if we are planning on explaining why the vote failed, we need to fix the other "vote failed" message as well.

On another note, I'd put the reason inside of brackets, this is not typographically correct :)
```py
(ctx, "Vote over {emoji} failed (not enough votes).").format(
```